sys/kern/src/arch/rv64/task.rs

```diff
@@ -20,17 +33,25 @@ use unwrap_lite::UnwrapLite;
 pub unsafe fn set_current_task(task: &task::Task) {
     // Safety: should be ok if the contract above is met
     // TODO: make me an atomic
```

Comment:
Out of date comment?

Reply:
Unchanged from source, suggest we address this as part of a wider comment scrub.